src/abbreviation/abbreviations.json

```diff
@@ -962,6 +962,8 @@
     "cap": "∩",
     "qed": "∎",
     "quad": " ",
+    "quot": "⧸",
+    "/": "⧸",
```

Comment:
I wonder if it's worth having `bigsolidus` too since that's the Unicode name

Reply:
Sure, why not:
```suggestion
    "bigsolidus": "⧸",
    "/": "⧸",
```